### ISSUE TITLE
optionally create dracut without zpool.cache

### ISSUE THREAD

**Issue by prometheanfire** [zfsonlinux/zfs#5399]
This is useful for creating shipable initrams

**Comment by kpande** [comment]
@prometheanfire how do you want this to work out? this is a local dracut.conf issue.

**Comment by prometheanfire** [comment]
Well, I don't think this is an issue anymore.  I'm using dracut without a zpool.cache at least.

**Comment by nabijaczleweli** [comment]
You're right: dracut will include dependencies of `zfs-import.target` – `zfs-import-{scan,cache}.service` by default – and not shipping `zpool.cache` will fall back to scan. This can be closed.
